python/src/puml/templates/object/object_diagram.puml.jinja2

```diff
@@ -1,10 +1,10 @@
-@startuml {{title}} Object Diagram
-title {{title}} Object Diagram
-{% for object in objects %}
+@startuml {{model.name}} Object Diagram
+title {{model.name}} Object Diagram
+{% for object in model.objects %}
 object {{object}}
 {% endfor %}
 
-{% for object_hierarchy in object_hierarchies %}
+{% for object_hierarchy in model.object_hierarchies %}
```

Comment:
Need to incorporate the blank line handling for jinja, there are some unneeded blanks between the object line and hierarchy data. 

There is syntax for inclusion with the loop notation to tell the jinja output to not inject lines for their entries. I believe it is a `-` at the trailing `%}`, but you should double check their documentation to confirm.

Reply:
Looking at the component template, the `-%}` is correct for helping remove empty lines due to template logic entries not pertaining to needed in place displayed output.